nixos/modules/misc/apropos.nix

```diff
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.documentation.apropos;
+
+in
+
+{
+
+  options.documentation.apropos = {
+      enable = mkOption {
```

Comment:
I think this whole config can be moved to `documentation.nix` under the `mkIf cfg.man.enable` without an additional `.enable` option if it can be done at build time.